src/context/AuthContext/index.tsx

```diff
@@ -3,30 +3,23 @@ import type { Provider } from '@wagmi/core';
 import config from 'config';
 import { Signer, getDefaultProvider } from 'ethers';
 import noop from 'noop-ts';
-import React, { useCallback, useContext, useState } from 'react';
+import React, { useCallback, useContext } from 'react';
 import { useTranslation } from 'translation';
 import { useAccount, useConnect, useDisconnect, useProvider, useSigner } from 'wagmi';
 
 import { Connector, connectorIdByName } from 'clients/web3';
 import { AuthModal } from 'components/AuthModal';
-import { LS_KEY_CONNECTED_CONNECTOR } from 'constants/localStorageKeys';
-import getConnectedConnector from 'context/AuthContext/getConnectedConnector';
 import useCopyToClipboard from 'hooks/useCopyToClipboard';
 import { isRunningInInfinityWalletApp } from 'utilities/walletDetection';
 
-export interface Account {
-  address: string;
-  connector?: Connector;
-}
```

Comment:
@therealemjy I also went ahead and refactored this out since the connector is not present anymore. This change leads to most of the other changes in this PR, since I believe it is just simpler for us to reach out the `accountAddress` directly, as it would be the only key in this object. But let me know if you think this is still useful and I can revert this.

Reply:
Good idea!